spanner/spanner_leaderboard/leaderboard_test.go

```diff
@@ -61,8 +74,8 @@ func TestSample(t *testing.T) {
 	runCommand := func(t *testing.T, cmd string, dbName string, timespan int) string {
 		t.Helper()
 		var b bytes.Buffer
-		// Set timeout to 60s so it should avoid DeadlineExceeded error.
-		cctx, cancel := context.WithTimeout(ctx, 60000*time.Millisecond)
+		// Set timeout to 600 seconds so it should avoid DeadlineExceeded error.
+		cctx, cancel := context.WithTimeout(ctx, 600*time.Second)
```

Comment:
Can we insert fewer rows into the table to speed it up? I _think_ this is only used in a Codelab. So, I'm willing to have a less-robust test for the sake of reliability and speed.

Reply:
I noticed that the test case is executing the `insertPlayers` and `insertScores` twice. AFAICT, there's no good reason for doing so, and reducing that to once for both methods will significantly reduce the total number of rows that are inserted (from 1600 to 400 for scores and from 200 to 100 for players). That should speed it up quite a bit.

The 10 minutes should be overkill in most cases. It seems that the deadline is most commonly exceeded if Spanner aborts the transaction a couple of times in a row, causing multiple retries.